Modules/Devices/src/Devices.Application/Devices/Commands/RegisterDevice/Handler.cs

```diff
@@ -31,12 +32,15 @@ public async Task<RegisterDeviceResponse> Handle(RegisterDeviceCommand command,
     {
         var identity = await _identitiesRepository.FindByAddress(_userContext.GetAddress(), cancellationToken, track: true) ?? throw new NotFoundException(nameof(Identity));
 
+        if (await _identitiesRepository.HasBackupDevice(identity.Address, cancellationToken))
+            throw new ApplicationException(ApplicationErrors.Devices.BackupDeviceAlreadyExists());
```

Comment:
So... now you're only able to create a new device if you don't have a backup device already. ;)